manifests/plugin/abrt.pp

```diff
@@ -33,7 +35,8 @@
 #
 class foreman_proxy::plugin::abrt (
   $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $version                 = undef,
```

Comment:
No worries.  It'd be nice to move away from params as we no longer require it for kafo, but there's often a lot of logic in there so mixing and matching is also messy.